### PR TITLE
fix:Connection Settings on login screen doesn't feel clickable.

### PR DESCRIPTION
**Pull request by singhalok641**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by singhalok641** [comment]
#525

**Comment by singhalok641** [comment]
#525

**Comment by singhalok641** [comment]
photo_2017-03-01_20-21-24

File: mifosng-android/src/main/res/layout/activity_login.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
 
             <TextView
```
Comment (therajanmaurya) [inline mifosng-android/src/main/res/layout/activity_login.xml]: Change TextVIew to Button

Comment (singhalok641) [inline mifosng-android/src/main/res/layout/activity_login.xml]: so I should create a new pull request or make changes in this one ?

Comment (therajanmaurya) [inline mifosng-android/src/main/res/layout/activity_login.xml]: Make the changes in this PR and  squash your commit and force push again.

Comment (singhalok641) [inline mifosng-android/src/main/res/layout/activity_login.xml]: cool

**Pull request closed without merging**